### PR TITLE
Convert to world time

### PR DESCRIPTION
**Pull request by shootdot**
**Converts local time to different time by inputting the timezones**
**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: actions/convert_to_world_time_MOD.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+module.exports = {
+  name: 'Convert To World Time',
+  section: 'Other Stuff',
+
+  subtitle: function (data) {
```
Comment (almeidx) [inline actions/convert_to_world_time_MOD.js:5]:
```suggestion
  subtitle (data) {
```

File: actions/convert_to_world_time_MOD.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    return 'Input a timezone and retrieve its current time.'
+  },
+
+  variableStorage: function (data, varType) {
```
Comment (almeidx) [inline actions/convert_to_world_time_MOD.js:9]:
```suggestion
  variableStorage (data, varType) {
```

File: actions/convert_to_world_time_MOD.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    const type = parseInt(data.storage)
+    if (type !== varType) return
+    const dataType = 'Time'
+    return ([data.varName, dataType])
```
Comment (almeidx) [inline actions/convert_to_world_time_MOD.js:13]:
```suggestion
    return ([data.varName, 'Time'])
```

File: actions/convert_to_world_time_MOD.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  },
+
+  fields: ['textbox', 'info', 'storage', 'varName'],
+
+  html: function (isEvent, data) {
```
Comment (almeidx) [inline actions/convert_to_world_time_MOD.js:18]:
```suggestion
  html (isEvent, data) {
```

File: actions/convert_to_world_time_MOD.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    return `
+
```
Comment (almeidx) [inline actions/convert_to_world_time_MOD.js:20]:
```suggestion
```

File: actions/convert_to_world_time_MOD.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+        Store Converted Time In:<br>
+        <select id="storage" class="round">
+            ${data.variables[1]}
+        </select>
+    </div>
+    <div id="varNameContainer" style="float: right; width: 60%;">
+        Variable Name:<br>
+        <input id="varName" class="round" type="text">
+    </div>
+</div>`
```
Comment (almeidx) [inline actions/convert_to_world_time_MOD.js:38]:
```suggestion
<div>
  <div style="width: 90%;">
    To Be Converted:<br>
    <input id="textbox" class="round" type="text">
  </div><br>
</div><br>
<div style="padding-top: 8px;">
  <div style="float: left; width: 35%;">
    Store Converted Time In:<br>
    <select id="storage" class="round">
      ${data.variables[1]}
    </select>
  </div>
  <div id="varNameContainer" style="float: right; width: 60%;">
    Variable Name:<br>
    <input id="varName" class="round" type="text">
  </div>
</div>`
```

File: actions/convert_to_world_time_MOD.js [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+  },
+
+  init: function () {},
```
Comment (almeidx) [inline actions/convert_to_world_time_MOD.js:41]:
```suggestion
  init () {},
```

File: actions/convert_to_world_time_MOD.js [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+  action: function (cache) {
```
Comment (almeidx) [inline actions/convert_to_world_time_MOD.js:43]:
```suggestion
  action (cache) {
```

File: actions/convert_to_world_time_MOD.js [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+    const Mods = this.getMods()
+    const moment = Mods.require('moment-timezone')
+    const str = this.evalMessage(data.textbox, cache)
+    const timec = moment().tz(str).format('dddd, MMMM Do YYYY, h:mm:ss a')
+    const storage = parseInt(data.storage)
+    const varName = this.evalMessage(data.varName, cache)
+    this.storeValue(timec, storage, varName, cache)
+    this.callNextAction(cache)
+  },
+  mod: function () { }
```
Comment (almeidx) [inline actions/convert_to_world_time_MOD.js:54]:
```suggestion

  mod () {}
```

File: actions/convert_to_world_time_MOD.js [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+    const Mods = this.getMods()
+    const moment = Mods.require('moment-timezone')
+    const str = this.evalMessage(data.textbox, cache)
+    const timec = moment().tz(str).format('dddd, MMMM Do YYYY, h:mm:ss a')
+    const storage = parseInt(data.storage)
+    const varName = this.evalMessage(data.varName, cache)
+    this.storeValue(timec, storage, varName, cache)
+    this.callNextAction(cache)
+  },
+  mod () { }
```
Comment (almeidx) [inline actions/convert_to_world_time_MOD.js:52]:
```suggestion
  },

  mod () {}
```

Comment (shootdot) [reply actions/convert_to_world_time_MOD.js]: @almeidx Changes Done

**Comment by shootdot** [comment]
Almedia now what should I do ?

**Comment by almeidx** [comment]
Add this to the docs

**Pull request merged**